### PR TITLE
sbc: 1.4 -> 1.5

### PR DESCRIPTION
**Pull request by gebner**
###### Motivation for this change

New upstream version.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by gebner** [comment]
Fails on arm :-(

File: pkgs/development/libraries/sbc/default.nix [hunk 1]
```diff
@@ -14,6 +15,7 @@ stdenv.mkDerivation rec {
   meta = with stdenv.lib; {
```
Comment (SuperSandro2000) [inline pkgs/development/libraries/sbc/default.nix:15]:
```suggestion
  meta = with lib; {
```

File: pkgs/development/libraries/sbc/default.nix [hunk 2]
```diff
@@ -1,11 +1,12 @@
...
+  version = "1.5";
 
   src = fetchurl {
-    url = "http://www.kernel.org/pub/linux/bluetooth/${name}.tar.xz";
-    sha256 = "1jal98pnrjkzxlkiqy0ykh4qmgnydz9bmsp1jn581p5kddpg92si";
+    url = "http://www.kernel.org/pub/linux/bluetooth/${pname}-${version}.tar.xz";
+    sha256 = "1liig5856crb331dps18mp0s13zbkv7yh007zqhq97m94fcddfhc";
   };
 
   nativeBuildInputs = [ pkgconfig ];
```
Comment (SuperSandro2000) [inline pkgs/development/libraries/sbc/default.nix:12]:
```suggestion
  nativeBuildInputs = [ pkg-config ];
```

File: pkgs/development/libraries/sbc/default.nix [hunk 3]
```diff
@@ -1,11 +1,12 @@
 { stdenv, fetchurl, pkgconfig, libsndfile }:
```
Comment (SuperSandro2000) [inline pkgs/development/libraries/sbc/default.nix:1]:
```suggestion
{ stdenv, fetchurl, pkg-config, libsndfile }:
```

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

**Comment by SuperSandro2000** [comment]
Please target master.

**Comment by wegank** [comment]
Superseded by #184673.

**Pull request closed without merging**